kuma/static/js/components/banners/utils/banners-state-util.js

```diff
@@ -0,0 +1,126 @@
+(function() {
+    'use strict';
+    /**
+     * Length of time a banner remains hidden after it's been disabled
+     * by clicking the close button.
+     * @const
+     */
+    var BANNER_DISABLED_EXPIRATION = 5 * 24 * 60 * 60 * 1000; // 5 days.
+    /**
+     * A simple string to use as a namespace for `localStorage` items
+     * @const
+     */
+    var LOCALSTORAGE_NAMESPACE = 'mdnBanner.';
+    var mdnBannersStateUtil = {
+        /**
+         * Filters the `bannerIds` down to only those in an active waffle flag state
+         * @returns {Array} of banner ids or an empty array if no active banners
+         */
+        getActiveWaffledBanners: function() {
+            var activeBannerIds = [];
+            var bannerIds = ['developer_needs', 'contrib_beta'];
```

Comment:
I'd make this ALL_BANNER_IDS up at the top of the file, so that when a new banner is added it is easy to find the line of code that needs updating...